.circleci/config.yml

```diff
@@ -0,0 +1,224 @@
+version: 2.1

```

Comment:
Build will need to be similar to one of our repos that are being published to NPM. (E.g. shared library, Foxden Tools, Version Controller client, etc.)